src/modules/generator.ts

```diff
@@ -19,7 +19,7 @@ const generatorModule = (dockerfileData: DockerfileData, options: HermitOptions)
   content += `FROM ${dockerfileData.images[0]} ${isMultiStage ? "AS build-env" : ""}\n`;
 
   // Import service source code
-  content += "ADD . /app\n";
+  content += "COPY . /app\n";
```

Comment:
Interesting. Why should we use `COPY` instead of `ADD`? Sorry, I forgot their differences 😅

Reply:
Using COPY instead of ADD is considered [good practice](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy)